views/forms/form_success.php

```diff
@@ -37,14 +37,17 @@
                 echo '<div id="referral_submit_success" class="alert alert-success" role="alert">';
                 echo '<h4 class="alert-heading">Success!</h4>';
                 if ($form_type == "agency referral") {
```

Comment:
I think this `if else` section of code could be reduced by just inserting `$form_type` into your `echo`s. 

`echo '<p>You have successfully submitted the ' . $form_type . ' form. </p><hr>';`